src/app/index.ts

```diff
@@ -265,64 +266,41 @@ module.exports = yo.extend({
   {
     return new Promise((resolve, reject) => {
       try {
-        let language = this.project.scriptType === typescript ? 'ts' : 'js';
-        const starterCode = generateStarterCode(this.project.host);
-        const templateFills = Object.assign({}, this.project, starterCode);
+        language = this.project.scriptType === typescript ? 'ts' : 'js';
         let jsonData = new projectsJsonData(this.templatePath());
         let projectRepoBranchInfo = jsonData.getProjectRepoAndBranch(this.project.projectType, language);
+        const templateFills = Object.assign({}, this.project);
```

Comment:
This declaration can be moved into the manifest-only else clause below. It is only used for that as far as I can tell.